code/datums/uplink/highly visible and dangerous weapons.dm

```diff
@@ -84,11 +84,21 @@
 	item_cost = 6
 	path = /obj/item/martial_manual
 
+/*
+
+//This is commented out incase somebody wants to easily add it back in
```

Comment:
No. If someone wants to add it back in they can code it back in. We don't leave commented out code.